Source/WebKit/UIProcess/AuxiliaryProcessProxy.cpp

```diff
@@ -378,6 +390,8 @@ void AuxiliaryProcessProxy::replyToPendingMessages()
 
 void AuxiliaryProcessProxy::shutDownProcess()
 {
+    throttler().didDisconnectFromProcess();
```

Comment:
Could we do this in a ScopeExit? This will release the assertion. However, `platformStartConnectionTerminationWatchdog()` below will take an assertion again. It'd be nice if the process didn't suspend and resume because we release the assertion and take a new one right after. We can do better in terms of ordering here.